tools/automator-main.sh

```diff
@@ -226,6 +234,7 @@ validate_opts() {
   if [ -z "${git_exclude:-}" ]; then
     git_exclude=""
   fi
+
```

Comment:
Why did you remove the check when `fetch_tags` is present, then `merge` must be `true`?

Reply:
Restored.